Source/Resources/Content/MSGraph/Groups/Baseline - Microsoft 365 Users.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$description": "| What does this do? | Creates a group that is used to assign configurations that should be applied to all Microsoft 365 users. |\n| Why should you use this? | If you want to apply certain Simeon Baseline configurations to your users. |\n| What is the end-user impact? | Users in this group will have the following configurations applied. |\n| [Sign-In Risk Policy](https://simeoncloud.github.io/docs/#/baseline?id=azure-ad-gt-security-gt-identity-protection), [User Risk and MFA Registration Policy](https://simeoncloud.github.io/docs/#/baseline?id=user-risk-and-mfa-registration-policy), [Password Reset Policy](https://simeoncloud.github.io/docs/#/baseline?id=azure-ad-gt-password-reset), [Baseline - Protect Policy Managed Client Apps on Unmanaged Android Devices](https://simeoncloud.github.io/docs/#/baseline?id=baseline-protect-policy-managed-client-apps-on-unmanaged-android-devices), [Baseline - Protect Policy Managed Client Apps on Unmanaged iOS Devices](https://simeoncloud.github.io/docs/#/baseline?id=baseline-protect-policy-managed-client-apps-on-unmanaged-ios-devices), [Baseline - Configure Policy Managed Client Apps on Unmanaged iOS Devices](https://simeoncloud.github.io/docs/#/baseline?id=baseline-configure-policy-managed-client-apps-on-unmanaged-ios-devices), [Baseline - Microsoft 365 Users - Preview](https://simeoncloud.github.io/docs/#/baseline?id=baseline-microsoft-365-users-preview), [Device Registration Policy](https://simeoncloud.github.io/docs/#/baseline?id=azure-ad-gt-device-settings) |\n| Learn more | [Assign policies to users and groups](https://docs.microsoft.com/en-us/microsoftteams/assign-policies-users-and-groups) |",  
+  "$description": "| What does this do? | Creates a group that is used to assign configurations that should be applied to all Microsoft 365 users. |\n| Why should you use this? | If you want to apply certain Simeon Baseline configurations to your users. |\n| What is the end-user impact? | Users in this group will have the following configurations applied. |\n| Sign-In Risk Policy, User Risk and MFA Registration Policy, Password Reset Policy, Baseline - Protect Policy Managed Client Apps on Unmanaged Android Devices, Baseline - Protect Policy Managed Client Apps on Unmanaged iOS Devices, Baseline - Configure Policy Managed Client Apps on Unmanaged iOS Devices, Baseline - Microsoft 365 Users - Preview, Device Registration Policy |\n| Learn more | [Assign policies to users and groups](https://docs.microsoft.com/en-us/microsoftteams/assign-policies-users-and-groups) |",

```

Comment:
I would put in a general statement that user policies in the baseline are assigned to this group. Same thing with corporate devices

Reply:
Made some changes. Take a look.

Reply:
Looks good to me